.github/workflows/release.yaml

```diff
@@ -110,3 +110,14 @@ jobs:
           projectName: ${{ secrets.CLOUDFLARE_PROJECT_NAME }}
           directory: build
           githubToken: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Upload source maps to Sentry
+        uses: getsentry/action-release@v1
```

Comment:
this should be versioned using a commit sha, not semver

Reply:
Do you mean a Github Action itself or a Sentry release? If former, any particular reason behind? Other Github actions are referenced by semver too.

If latter, Sentry uses commit sha by default, so no need to set a special release name (https://github.com/marketplace/actions/sentry-release#parameters - see "version")

Reply:
former, for security purposes. v1 can change what it does at the publisher's whim. using a sha is more stable.

other github actions should also be referenced by sha...eventually.

Reply:
Okay, makes sense. Overall I agree, I will create a ticket for updating this and other versions too at some point. Meanwhile, will apply that to this particular action.